hca/added_command.py

```diff
@@ -3,7 +3,7 @@
 
 import requests
 
-from constants import Constants
```

Comment:
Got it. Should I refactor this diff and make a new one for that or just keep that in mind for the next time?